examples/usecases/llm_diffusion_serving_app/Readme.md

```diff
@@ -0,0 +1,126 @@
+ => => writing image sha256:e900f12e6dad3ec443966766f82860427fa066aefe504a415eecf69bf4c3c043                                                                                                               0.0s
+ => => naming to docker.io/pytorch/torchserve:llm_diffusion_serving_app                                                                                                                                    0.0s
+
+Run the following command to start the Multi-image generation App
+
+docker run --rm -it --platform linux/amd64 \
+-p 127.0.0.1:8080:8080 \
+-p 127.0.0.1:8081:8081 \
+-p 127.0.0.1:8082:8082 \
+-p 127.0.0.1:8084:8084 \
+-p 127.0.0.1:8085:8085 \
+-v /home/ubuntu/serve/model-store-local:/home/model-server/model-store \
+-e MODEL_NAME_LLM=meta-llama/Llama-3.2-3B-Instruct \
+-e MODEL_NAME_SD=stabilityai/stable-diffusion-xl-base-1.0  \
+pytorch/torchserve:llm_diffusion_serving_app
+
+Note: You can replace the model identifier as needed
+```
+
+## What to expect
+Once you launch using the the docker run cmd, it launches two streamlit apps:
+1. TorchServe Server App (running at http://localhost:8084) to start/stop TorchServe, load/register models, scale up/down workers. 
+2. Client App (running at http://localhost:8085) where you can enter prompt for Image generation. 
+
+#### Sample Output:
+
+```console
+ubuntu@ip-10-0-0-137:~/serve$ docker run --rm -it --platform linux/amd64 \
+-p 127.0.0.1:8080:8080 \
+-p 127.0.0.1:8081:8081 \
+-p 127.0.0.1:8082:8082 \
+-p 127.0.0.1:8084:8084 \
+-p 127.0.0.1:8085:8085 \
+-v /home/ubuntu/serve/model-store-local:/home/model-server/model-store \
+-e MODEL_NAME_LLM=meta-llama/Llama-3.2-3B-Instruct \
+-e MODEL_NAME_SD=stabilityai/stable-diffusion-xl-base-1.0  \
+pytorch/torchserve:llm_diffusion_serving_app
+
+Preparing meta-llama/Llama-3.2-1B-Instruct
+/home/model-server/llm_diffusion_serving_app/llm /home/model-server/llm_diffusion_serving_app
+Model meta-llama---Llama-3.2-1B-Instruct already downloaded.
+Model archive for meta-llama---Llama-3.2-1B-Instruct exists.
+/home/model-server/llm_diffusion_serving_app
+
+Preparing stabilityai/stable-diffusion-xl-base-1.0
+/home/model-server/llm_diffusion_serving_app/sd /home/model-server/llm_diffusion_serving_app
+Model stabilityai/stable-diffusion-xl-base-1.0 already downloaded
+Model archive for stabilityai---stable-diffusion-xl-base-1.0 exists.
+/home/model-server/llm_diffusion_serving_app
+
+Collecting usage statistics. To deactivate, set browser.gatherUsageStats to false.
+
+
+Collecting usage statistics. To deactivate, set browser.gatherUsageStats to false.
+
+
+  You can now view your Streamlit app in your browser.
+
+  Local URL: http://localhost:8085
+  Network URL: http://123.11.0.2:8085
+  External URL: http://123.123.12.34:8085
+
+
+  You can now view your Streamlit app in your browser.
+
+  Local URL: http://localhost:8084
+  Network URL: http://123.11.0.2:8084
+  External URL: http://123.123.12.34:8084
+```
```

Comment:
Can we please include screenshots of what the UI looks like for both server & client, an example of images with prompt